lib/linguist/shebang.rb

```diff
@@ -10,10 +10,11 @@ class Shebang
     #
     # Returns an Array with one Language if the blob has a shebang with a valid
```

Comment:
We should change the comment here too to reflect the change in behaviour.

Reply:
I update this comment and added one for both the Filename and the Extension strategies. The wording's not great, so if you have any suggestion, I'll be happy to update!